efficientdet/anchors.py

```diff
@@ -183,6 +216,75 @@ def _generate_anchor_boxes(image_size, anchor_scale, anchor_configs):
   anchor_boxes = np.vstack(boxes_all)
   return anchor_boxes
 
+def _generate_detections_tf(cls_outputs, box_outputs, anchor_boxes, indices,
+                         classes, image_id, image_scale, num_classes, use_native_nms=False):
```

Comment:
Format it to make them align. This line is also too long (please make it <80 characters).